ClientTests/MockProfile.swift

```diff
@@ -88,6 +88,10 @@ public class MockProfile: Profile {
     }()
 
     lazy var bookmarks: protocol<BookmarksModelFactory, ShareToDestination> = {
+        // Make sure the rest of our tables are initialized before we try to read them!
+        // This expression is for side-effects only.
+        let p = self.places
```

Comment:
So how does this fix work?

If `places` lazy initialization is sync, why does it matter when we do it?

If `places` lazy initialization is async, doesn't that mean there's still a race?